pom.xml

```diff
@@ -235,9 +235,53 @@
         -->
 
         <!-- Please set the appropriate version according to the web browser. -->
```

Comment:
- [ ] Write it after UT library definition and change it to `<! -- == Begin E2E Test == -->`. (The same also applies to End)

Reply:
Fixed where selenium library dependency definition is defined.
cd906ab